frontend/packages/component-library/src/cms/header/header.module.scss

```diff
@@ -0,0 +1,323 @@
+      text-decoration: none;
+      padding: 1rem;
+      border-inline: 2px solid transparent;
+
+      @include mixins.transition-all;
+
+      &:hover,
+      &:active {
+        background-color: var(--background-brand-teal-strong);
+      }
+
+      &:focus-visible {
+        border-inline: 2px solid var(--neutral-white-alpha-70);
+        outline: none;
+      }
+    }
+  }
+}
+
+// Button links styles
+.buttonLinks {
+  position: relative;
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+
+  .signIn.mainMenu,
+  .newProject {
+    background: transparent;
+
+    &:hover {
+      background: var(--background-brand-teal-strong);
+    }
+  }
+
+  .createAccount.mainMenu,
+  .goToDashboard.mainMenu {
+    color: var(--text-brand-teal-primary-fixed);
+
+    &:hover {
+      background: var(--neutral-white-alpha-90);
+    }
+  }
+
+  button,
+  a {
+    &:focus-visible {
+      outline-color: var(--neutral-white-alpha-70);
+    }
+
+    span {
+      font-weight: 500;
+    }
+  }
+}
+
+// Icon button styles
+.buttonLinks .iconButton {
+  padding: 0;
+  background: transparent;
+  min-width: unset;
+  width: 1.75rem;
+  height: 1.75rem;
+  border-radius: 100%;
+
+  & > i {
+    font-size: 1.375rem;
+  }
+
+  &:hover,
+  &:focus {
+    color: var(--neutral-white-alpha-90);
+    background: transparent;
+  }
+
+  &.helpButton {
+    margin-inline: 0.25rem -0.5rem;
+
+    [dir='rtl'] & {
+      transform: scaleX(-1);
+    }
+  }
+
+  &.hamburgerButton {
+    margin-inline-start: -0.25rem;
+  }
+}
+
+// Menu styles
+.menu {
+  @include mixins.border-styles;
+  list-style: none;
+  position: absolute;
+  top: 3rem;
+  padding: 0.375rem;
+  margin: 0;
+  min-width: 228px;
+  max-width: 300px;
+  background: var(--background-neutral-primary);
+  box-shadow: $box-shadow;
+  display: flex;
+  flex-direction: column;
+  text-align: initial;
+
+  a {
+    @include typography.body-three;
+    font-weight: 500;
+    display: block;
+    margin-bottom: 0;
+    text-decoration: none;
+    padding: 0.5rem 0.75rem;
+
+    @include mixins.transition-all;
+
+    &:hover {
+      border-radius: variables.$regular-border-radius;
+      background: var(--background-neutral-tertiary);
+    }
+
+    &:focus-visible {
+      @include mixins.focus-styles;
+      outline-offset: 0;
+      border-radius: variables.$regular-border-radius;
+    }
+  }
+}
+
+// New Project menu styles
+.newProjectMenu {
+  a {
+    margin: 0;
+    padding-inline: 0.375rem;
+    display: flex;
+    flex-direction: row;
+    align-items: start;
+    gap: 0.625rem;
+
+    figure {
+      width: 2.5rem;
+    }
+
+    & > div {
+      flex: 2;
+
+      .description {
+        font-size: 0.75rem;
+        margin: 0;
+        color: var(--text-neutral-tertiary);
+      }
+    }
+  }
+
+  li:last-child > a {
+    align-items: center;
+  }
+}
+
+// Account links styles
+.accountLinks {
+  display: flex;
+  gap: 0.5rem;
+
+  [dir='rtl'] & .goToDashboard i {
+    transform: scaleX(-1);
```

Comment:
super nit: are we sure we need this? (won't regular change of content order be enough?)

Reply:
We still need this, the content order changes but it doesn't flip the icon itself (same is true for the other comment).